.github/workflows/test-release.yml

```diff
@@ -69,3 +73,14 @@ jobs:
 
           cml-send-comment report.md
           cml-send-github-check report.md
+  release:
+    needs: test
+    runs-on: ubuntu-latest
+    if: github.event_name == 'release'
```

Comment:
Actions seems to be stuck also

Reply:
> _I would prefer when merging into master to make it much more automatised._
> _Updating the package version and merging into master should do the trick_

🤷🏼‍♂️  It's a minuscule project, so... why not? It would save us a few seconds, and tests plus branch protection should be enough to avoid breaking the `v1` tag.

👋🏼🪔🧞‍♂️ @casperdcl, any preference from your CI/CD magic lamp?

Reply:
The [tagging approach for actions](https://docs.github.com/en/actions/creating-actions/about-actions#using-tags-for-release-management) is very different from tags on normal repos.

- https://github.com/Actions-R-Us/actions-tagger is rubbish
- `git push origin :refs/tags/{tag} && git tag --force {tag} && git push origin {main} --tags` is also rubbish

You need `git tag v1.x.x && git tag -f v1 && git push --tags -f`

I would suggest:

- closing this PR unmerged
- rename `master` to `v1` and use a hybrid of
  + [branch apprach for actions](https://docs.github.com/en/actions/creating-actions/about-actions#using-branches-for-release-management) for major (i.e. `@v1`, `@v2`) pointers
  + tags for semver (i.e. `v1.x.x`) pointers
    * ideally start writing release notes for these but accept that no `workflow.yml` file can enforce this